### PR TITLE
Project Setup With Webpack

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following:

- Setup github workflows
- Created .gitignore file and added node_modules
- Added linters for HTML, CSS and JavaScript
- Added jest for unit tests
- Setup webpack and its configuration

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import _ from 'lodash';
```
Comment (ismayito) [inline src/index.js]: Hello, thanks for the work done so far, but I propose that lodash is removed as is not required as of yet.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import instagramBrands from './assets/instagram-brands.svg';
+
+function component() {
+  const element = document.createElement('div');
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+  return element;
+}
+
```
Comment (ismayito) [inline src/index.js]: Even this lodash sample script should be removed because we are more likely not to use it

File: src/sum.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function sum(a, b) {
+  return a + b;
+}
```
Comment (ismayito) [inline src/sum.js]: I do not know what this sum function is being used for. You can explain please, If it is not required then we can do away with it at the moment.

File: src/sum.test.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import sum from './sum.js';
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
```
Comment (ismayito) [inline src/sum.test.js]: I think all this jest test sample example can be removed as of now.

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
```
Comment (ismayito) [inline src/index.html:6]: I propose that title of our document is changed to be the tittle of our project application

**Review by ismayito — COMMENTED** [review]
Hi @Chimoneg27  thanks for the great work done in the project set up, However there are some changes we can make to progress on 👏👏👏
**_CHANGES REQUIRED_**
()[https://media.tenor.com/1-Kd_C4W8qUAAAAM/youre-gonna-change-some-stuff-ok-chris-rock.gif]

File: src/sum.js (hunk 3 above)
Comment (Chimoneg27) [reply src/sum.js]: These are just dummy tests for Jest. So we'll remove them later

**Review by ismayito — APPROVED** [review]
Hello Garvin thanks for making the suggested changes, the project is now complete.\
There is nothing else to say apart from you merging it and you move forward.

**Pull request merged**